client/src/Button.js

```diff
@@ -1,11 +1,45 @@
 import React from "react";
```

Comment:
All the logic you've added looks great, I think we just need to move it over to the `LoginSignUp.js` component, instead of the Button component. We want to keep each component doing its own job. 

Next you'll include `LoginSignUp` in our App.js file, and can add it in as <LoginSignUp />